qutebrowser/app.py

```diff
@@ -255,7 +256,11 @@ def process_pos_args(args, via_ipc=False, cwd=None):
             log.init.debug("Empty argument")
             win_id = mainwindow.get_window(via_ipc, force_window=True)
         else:
-            win_id = mainwindow.get_window(via_ipc)
+            if via_ipc and target_arg and target_arg != 'auto':
```

Comment:
This happens because the slot is specified with a `str` type as argument, which means the Python string will be converted to a `QString`. For historical reasons, [there's a difference between empty and null QStrings](http://doc.qt.io/qt-5/qstring.html#distinction-between-null-and-empty-strings). What happens here is that PyQt converts `None` to a null `QString`, and later converts it back to an empty Python string (because Python strings don't have this distinction, and a null string is also empty).

Can you please add the `target_arg` argument to the docstring here and add a quick explanation of what's happening?